gas-report.txt

```diff
@@ -5,13 +5,13 @@
 ·············|············|·············|··············|·············|···············|··············
 |  Contract  ·  Method    ·  Min        ·  Max         ·  Avg        ·  # calls      ·  usd (avg)  │
 ·············|············|·············|··············|·············|···············|··············
-|  FundMe    ·  fund      ·      91289  ·      108389  ·      99839  ·           10  ·          -  │
+|  FundMe    ·  fund      ·      91501  ·      108601  ·     100051  ·           10  ·          -  │
```

Comment:
How is this related to the PR? You should probably figure out how to remove it 🙂